iib/workers/config.py

```diff
@@ -187,14 +202,18 @@ def validate_celery_config(conf, **kwargs):
 
     _validate_iib_org_customizations(conf['iib_organization_customizations'])
 
-    iib_request_logs_dir = conf.get('iib_request_logs_dir')
-    if iib_request_logs_dir:
-        if not os.path.isdir(iib_request_logs_dir):
-            raise ConfigError(
-                f'iib_request_logs_dir, {iib_request_logs_dir}, must exist and be a directory'
-            )
-        if not os.access(iib_request_logs_dir, os.W_OK):
-            raise ConfigError(f'iib_request_logs_dir, {iib_request_logs_dir}, is not writable!')
+    for directory in ('iib_request_logs_dir', 'iib_request_related_bundles_dir'):
+        iib_request_temp_data_dir = conf.get(directory)
+        if iib_request_temp_data_dir:
+            if not os.path.isdir(iib_request_temp_data_dir):
```

Comment:
Through what mechanism are these directories created? Isn't it better to create them instead of raising an error?

Reply:
I think the approach we have taken is that this must be pre-configured. In our case this is done via ansible. I like the current approach because it forces us to ensure these directories are created with the proper access.